bin/init.bash

```diff
@@ -99,7 +117,13 @@ generate_base_wc_config() {
     if [[ -f $file ]]; then
         yq merge "$tmpfile" "$file" --inplace -a=overwrite --overwrite --prettyPrint
     fi
+
     cat "$tmpfile" > "$file"
+
+      add_nfs_server_ip "$file" '.workload_cluster.nfs_ip_addresses.nfs.private_ip'
+
+
+
```

Comment:
Maybe remove at least the last 3 blank lines, and reindent the `add_nfs_server_ip` line?